Synthesis failed for Google.Cloud.Asset.V1

Hello! Autosynth couldn't regenerate Google.Cloud.Asset.V1. :broken_heart:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Asset.V1 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
witched to branch 'autosynth-Google.Cloud.Asset.V1'
2021-09-03 22:37:35,152 autosynth [DEBUG] > Running: git rev-parse --show-toplevel
2021-09-03 22:37:35,155 autosynth [DEBUG] > Running: git log -1 --pretty=%H
2021-09-03 22:37:35,158 autosynth [DEBUG] > Running: git remote get-url origin
2021-09-03 22:37:35,602 autosynth [DEBUG] > Running: git log 9c7eb1f9a78060a15cc309ab51ee0600c2e6f4f7..HEAD --pretty=%H --no-decorate
2021-09-03 22:37:35,605 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9c7eb1f9a78060a15cc309ab51ee0600c2e6f4f7
2021-09-03 22:37:35,608 autosynth [DEBUG] > Running: git log -1 --pretty=%at 31d2d3432d9a0b910d7858eab7720b63445f23b2
2021-09-03 22:37:35,611 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5d91687b183a6f2269ab75344e338f6bc584f5c4
2021-09-03 22:37:35,613 autosynth [DEBUG] > Running: git log -1 --pretty=%at bf57b2c88187c599bf669005d1d2ce80a40780df
2021-09-03 22:37:35,616 autosynth [DEBUG] > Running: git log -1 --pretty=%at b64d30b2b245cb805d1aa328e50258fdff6e628e
2021-09-03 22:37:35,619 autosynth [DEBUG] > Running: git checkout b71303f6e98ca6a6551bdb8abb433bb41425be9c
Note: switching to 'b71303f6e98ca6a6551bdb8abb433bb41425be9c'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at b71303f6e feat: Release of relationships in v1, Add content type Relationship to support relationship search Committer: yuwangyw@
2021-09-03 22:37:35,639 autosynth [DEBUG] > Running: git checkout b64d30b2b245cb805d1aa328e50258fdff6e628e
Note: switching to 'b64d30b2b245cb805d1aa328e50258fdff6e628e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at b64d30b2 chore: regenerate API index
2021-09-03 22:37:35,650 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Asset.V1-5
2021-09-03 22:37:35,653 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1-5
Switched to branch 'autosynth-Google.Cloud.Asset.V1-5'
2021-09-03 22:37:35,672 autosynth [INFO] > Running synthtool
2021-09-03 22:37:35,672 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-09-03 22:37:35,673 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Asset.V1/5/sponge_log.log
2021-09-03 22:37:35,674 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-09-03 22:37:35,880 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py.
On branch autosynth-Google.Cloud.Asset.V1-5
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Asset.V1 for compatibility checking
Generating Google.Cloud.Asset.V1
--gapic_out: System.InvalidOperationException: No resource type with name: 'compute.googleapis.com/Instance' for field ListInventoriesRequest.parent
   at Google.Api.Generator.ProtoUtils.ResourceDetails.LoadResourceReference(MessageDescriptor msgDesc, FieldDescriptor fieldDesc, IReadOnlyDictionary`2 resourcesByUrt, IReadOnlyDictionary`2 resourcesByParentComparison)+MoveNext() in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ResourceDetails.cs:line 288
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Google.Api.Generator.ProtoUtils.ProtoCatalog.<>c__DisplayClass0_2.<.ctor>b__20(FieldDescriptor field) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ProtoCatalog.cs:line 43
   at System.Linq.Enumerable.SelectIListIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector)
   at Google.Api.Generator.ProtoUtils.ProtoCatalog..ctor(String defaultPackage, IEnumerable`1 descs, IEnumerable`1 commonResourcesConfigs) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ProtoCatalog.cs:line 39
   at Google.Api.Generator.CodeGenerator.Generate(IReadOnlyList`1 descriptorProtos, IEnumerable`1 filesToGenerate, IClock clock, String grpcServiceConfigPath, IEnumerable`1 commonResourcesConfigPaths, Boolean generateMetadata)+MoveNext() in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/CodeGenerator.cs:line 113
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at Google.Protobuf.Collections.RepeatedField`1.AddRange(IEnumerable`1 values)
   at Google.Protobuf.Collections.RepeatedField`1.Add(IEnumerable`1 values)
   at Google.Api.Generator.Program.GenerateFromProtoc(Stream stdin, Stream stdout) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/Program.cs:line 174
rm: missing operand
Try 'rm --help' for more information.
2021-09-03 22:37:41,353 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Asset.V1:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Asset.V1')' returned non-zero exit status 1.
2021-09-03 22:37:41,378 autosynth [ERROR] > Synthesis failed
2021-09-03 22:37:41,378 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at b71303f6e feat: Release of relationships in v1, Add content type Relationship to support relationship search Committer: yuwangyw@
2021-09-03 22:37:41,404 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1
Switched to branch 'autosynth-Google.Cloud.Asset.V1'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/1febaa46-05f2-4ed4-bc64-3868959e8df0/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

Autosynth is still having trouble generating Google.Cloud.Asset.V1. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Asset.V1 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
osynth-Google.Cloud.Asset.V1
2021-09-04 08:37:17,192 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1
Switched to branch 'autosynth-Google.Cloud.Asset.V1'
2021-09-04 08:37:17,346 autosynth [DEBUG] > Running: git rev-parse --show-toplevel
2021-09-04 08:37:17,349 autosynth [DEBUG] > Running: git log -1 --pretty=%H
2021-09-04 08:37:17,353 autosynth [DEBUG] > Running: git remote get-url origin
2021-09-04 08:37:17,934 autosynth [DEBUG] > Running: git log 9c7eb1f9a78060a15cc309ab51ee0600c2e6f4f7..HEAD --pretty=%H --no-decorate
2021-09-04 08:37:17,937 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9c7eb1f9a78060a15cc309ab51ee0600c2e6f4f7
2021-09-04 08:37:17,939 autosynth [DEBUG] > Running: git log -1 --pretty=%at 31d2d3432d9a0b910d7858eab7720b63445f23b2
2021-09-04 08:37:17,942 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5d91687b183a6f2269ab75344e338f6bc584f5c4
2021-09-04 08:37:17,945 autosynth [DEBUG] > Running: git log -1 --pretty=%at bf57b2c88187c599bf669005d1d2ce80a40780df
2021-09-04 08:37:17,948 autosynth [DEBUG] > Running: git log -1 --pretty=%at b64d30b2b245cb805d1aa328e50258fdff6e628e
2021-09-04 08:37:17,951 autosynth [DEBUG] > Running: git checkout 7a79d5a97cae55043a6a62532e5c4bcfd0512a98
Note: switching to '7a79d5a97cae55043a6a62532e5c4bcfd0512a98'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 7a79d5a97 feat: add OSConfigZonalService API Committer: @jaiminsh
2021-09-04 08:37:17,971 autosynth [DEBUG] > Running: git checkout b64d30b2b245cb805d1aa328e50258fdff6e628e
Note: switching to 'b64d30b2b245cb805d1aa328e50258fdff6e628e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at b64d30b2 chore: regenerate API index
2021-09-04 08:37:17,983 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Asset.V1-5
2021-09-04 08:37:17,985 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1-5
Switched to branch 'autosynth-Google.Cloud.Asset.V1-5'
2021-09-04 08:37:18,005 autosynth [INFO] > Running synthtool
2021-09-04 08:37:18,005 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-09-04 08:37:18,005 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Asset.V1/5/sponge_log.log
2021-09-04 08:37:18,007 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-09-04 08:37:18,212 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py.
On branch autosynth-Google.Cloud.Asset.V1-5
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Asset.V1 for compatibility checking
Generating Google.Cloud.Asset.V1
--gapic_out: System.InvalidOperationException: No resource type with name: 'compute.googleapis.com/Instance' for field ListInventoriesRequest.parent
   at Google.Api.Generator.ProtoUtils.ResourceDetails.LoadResourceReference(MessageDescriptor msgDesc, FieldDescriptor fieldDesc, IReadOnlyDictionary`2 resourcesByUrt, IReadOnlyDictionary`2 resourcesByParentComparison)+MoveNext() in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ResourceDetails.cs:line 288
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Google.Api.Generator.ProtoUtils.ProtoCatalog.<>c__DisplayClass0_2.<.ctor>b__20(FieldDescriptor field) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ProtoCatalog.cs:line 43
   at System.Linq.Enumerable.SelectIListIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector)
   at Google.Api.Generator.ProtoUtils.ProtoCatalog..ctor(String defaultPackage, IEnumerable`1 descs, IEnumerable`1 commonResourcesConfigs) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ProtoCatalog.cs:line 39
   at Google.Api.Generator.CodeGenerator.Generate(IReadOnlyList`1 descriptorProtos, IEnumerable`1 filesToGenerate, IClock clock, String grpcServiceConfigPath, IEnumerable`1 commonResourcesConfigPaths, Boolean generateMetadata)+MoveNext() in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/CodeGenerator.cs:line 113
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at Google.Protobuf.Collections.RepeatedField`1.AddRange(IEnumerable`1 values)
   at Google.Protobuf.Collections.RepeatedField`1.Add(IEnumerable`1 values)
   at Google.Api.Generator.Program.GenerateFromProtoc(Stream stdin, Stream stdout) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/Program.cs:line 174
rm: missing operand
Try 'rm --help' for more information.
2021-09-04 08:37:23,764 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Asset.V1:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Asset.V1')' returned non-zero exit status 1.
2021-09-04 08:37:23,790 autosynth [ERROR] > Synthesis failed
2021-09-04 08:37:23,790 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7a79d5a97 feat: add OSConfigZonalService API Committer: @jaiminsh
2021-09-04 08:37:23,816 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1
Switched to branch 'autosynth-Google.Cloud.Asset.V1'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f5657c7c-7794-4ec1-a8b2-af18d25537d7/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

Autosynth is still having trouble generating Google.Cloud.Asset.V1. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Asset.V1 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
osynth-Google.Cloud.Asset.V1
2021-09-04 22:38:42,407 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1
Switched to branch 'autosynth-Google.Cloud.Asset.V1'
2021-09-04 22:38:42,572 autosynth [DEBUG] > Running: git rev-parse --show-toplevel
2021-09-04 22:38:42,576 autosynth [DEBUG] > Running: git log -1 --pretty=%H
2021-09-04 22:38:42,581 autosynth [DEBUG] > Running: git remote get-url origin
2021-09-04 22:38:43,075 autosynth [DEBUG] > Running: git log 9c7eb1f9a78060a15cc309ab51ee0600c2e6f4f7..HEAD --pretty=%H --no-decorate
2021-09-04 22:38:43,079 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9c7eb1f9a78060a15cc309ab51ee0600c2e6f4f7
2021-09-04 22:38:43,082 autosynth [DEBUG] > Running: git log -1 --pretty=%at 31d2d3432d9a0b910d7858eab7720b63445f23b2
2021-09-04 22:38:43,086 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5d91687b183a6f2269ab75344e338f6bc584f5c4
2021-09-04 22:38:43,089 autosynth [DEBUG] > Running: git log -1 --pretty=%at bf57b2c88187c599bf669005d1d2ce80a40780df
2021-09-04 22:38:43,092 autosynth [DEBUG] > Running: git log -1 --pretty=%at b64d30b2b245cb805d1aa328e50258fdff6e628e
2021-09-04 22:38:43,096 autosynth [DEBUG] > Running: git checkout 7a79d5a97cae55043a6a62532e5c4bcfd0512a98
Note: switching to '7a79d5a97cae55043a6a62532e5c4bcfd0512a98'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 7a79d5a97 feat: add OSConfigZonalService API Committer: @jaiminsh
2021-09-04 22:38:43,121 autosynth [DEBUG] > Running: git checkout b64d30b2b245cb805d1aa328e50258fdff6e628e
Note: switching to 'b64d30b2b245cb805d1aa328e50258fdff6e628e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at b64d30b2 chore: regenerate API index
2021-09-04 22:38:43,136 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Asset.V1-5
2021-09-04 22:38:43,140 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1-5
Switched to branch 'autosynth-Google.Cloud.Asset.V1-5'
2021-09-04 22:38:43,164 autosynth [INFO] > Running synthtool
2021-09-04 22:38:43,165 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-09-04 22:38:43,165 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Asset.V1/5/sponge_log.log
2021-09-04 22:38:43,167 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-09-04 22:38:43,415 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py.
On branch autosynth-Google.Cloud.Asset.V1-5
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Asset.V1 for compatibility checking
Generating Google.Cloud.Asset.V1
--gapic_out: System.InvalidOperationException: No resource type with name: 'compute.googleapis.com/Instance' for field ListInventoriesRequest.parent
   at Google.Api.Generator.ProtoUtils.ResourceDetails.LoadResourceReference(MessageDescriptor msgDesc, FieldDescriptor fieldDesc, IReadOnlyDictionary`2 resourcesByUrt, IReadOnlyDictionary`2 resourcesByParentComparison)+MoveNext() in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ResourceDetails.cs:line 288
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Google.Api.Generator.ProtoUtils.ProtoCatalog.<>c__DisplayClass0_2.<.ctor>b__20(FieldDescriptor field) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ProtoCatalog.cs:line 43
   at System.Linq.Enumerable.SelectIListIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector)
   at Google.Api.Generator.ProtoUtils.ProtoCatalog..ctor(String defaultPackage, IEnumerable`1 descs, IEnumerable`1 commonResourcesConfigs) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ProtoCatalog.cs:line 39
   at Google.Api.Generator.CodeGenerator.Generate(IReadOnlyList`1 descriptorProtos, IEnumerable`1 filesToGenerate, IClock clock, String grpcServiceConfigPath, IEnumerable`1 commonResourcesConfigPaths, Boolean generateMetadata)+MoveNext() in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/CodeGenerator.cs:line 113
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at Google.Protobuf.Collections.RepeatedField`1.AddRange(IEnumerable`1 values)
   at Google.Protobuf.Collections.RepeatedField`1.Add(IEnumerable`1 values)
   at Google.Api.Generator.Program.GenerateFromProtoc(Stream stdin, Stream stdout) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/Program.cs:line 174
rm: missing operand
Try 'rm --help' for more information.
2021-09-04 22:38:49,705 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Asset.V1:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Asset.V1')' returned non-zero exit status 1.
2021-09-04 22:38:49,740 autosynth [ERROR] > Synthesis failed
2021-09-04 22:38:49,740 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7a79d5a97 feat: add OSConfigZonalService API Committer: @jaiminsh
2021-09-04 22:38:49,772 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1
Switched to branch 'autosynth-Google.Cloud.Asset.V1'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4ce78ebb-25f3-423e-ad9d-440660ea8bc3/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

Autosynth is still having trouble generating Google.Cloud.Asset.V1. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Asset.V1 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
osynth-Google.Cloud.Asset.V1
2021-09-05 08:37:29,612 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1
Switched to branch 'autosynth-Google.Cloud.Asset.V1'
2021-09-05 08:37:29,793 autosynth [DEBUG] > Running: git rev-parse --show-toplevel
2021-09-05 08:37:29,796 autosynth [DEBUG] > Running: git log -1 --pretty=%H
2021-09-05 08:37:29,800 autosynth [DEBUG] > Running: git remote get-url origin
2021-09-05 08:37:30,249 autosynth [DEBUG] > Running: git log 9c7eb1f9a78060a15cc309ab51ee0600c2e6f4f7..HEAD --pretty=%H --no-decorate
2021-09-05 08:37:30,253 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9c7eb1f9a78060a15cc309ab51ee0600c2e6f4f7
2021-09-05 08:37:30,256 autosynth [DEBUG] > Running: git log -1 --pretty=%at 31d2d3432d9a0b910d7858eab7720b63445f23b2
2021-09-05 08:37:30,259 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5d91687b183a6f2269ab75344e338f6bc584f5c4
2021-09-05 08:37:30,262 autosynth [DEBUG] > Running: git log -1 --pretty=%at bf57b2c88187c599bf669005d1d2ce80a40780df
2021-09-05 08:37:30,265 autosynth [DEBUG] > Running: git log -1 --pretty=%at b64d30b2b245cb805d1aa328e50258fdff6e628e
2021-09-05 08:37:30,268 autosynth [DEBUG] > Running: git checkout 7a79d5a97cae55043a6a62532e5c4bcfd0512a98
Note: switching to '7a79d5a97cae55043a6a62532e5c4bcfd0512a98'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 7a79d5a97 feat: add OSConfigZonalService API Committer: @jaiminsh
2021-09-05 08:37:30,289 autosynth [DEBUG] > Running: git checkout b64d30b2b245cb805d1aa328e50258fdff6e628e
Note: switching to 'b64d30b2b245cb805d1aa328e50258fdff6e628e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at b64d30b2 chore: regenerate API index
2021-09-05 08:37:30,303 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Asset.V1-5
2021-09-05 08:37:30,306 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1-5
Switched to branch 'autosynth-Google.Cloud.Asset.V1-5'
2021-09-05 08:37:30,327 autosynth [INFO] > Running synthtool
2021-09-05 08:37:30,327 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-09-05 08:37:30,327 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Asset.V1/5/sponge_log.log
2021-09-05 08:37:30,329 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-09-05 08:37:30,545 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py.
On branch autosynth-Google.Cloud.Asset.V1-5
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Asset.V1 for compatibility checking
Generating Google.Cloud.Asset.V1
--gapic_out: System.InvalidOperationException: No resource type with name: 'compute.googleapis.com/Instance' for field ListInventoriesRequest.parent
   at Google.Api.Generator.ProtoUtils.ResourceDetails.LoadResourceReference(MessageDescriptor msgDesc, FieldDescriptor fieldDesc, IReadOnlyDictionary`2 resourcesByUrt, IReadOnlyDictionary`2 resourcesByParentComparison)+MoveNext() in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ResourceDetails.cs:line 288
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Google.Api.Generator.ProtoUtils.ProtoCatalog.<>c__DisplayClass0_2.<.ctor>b__20(FieldDescriptor field) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ProtoCatalog.cs:line 43
   at System.Linq.Enumerable.SelectIListIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector)
   at Google.Api.Generator.ProtoUtils.ProtoCatalog..ctor(String defaultPackage, IEnumerable`1 descs, IEnumerable`1 commonResourcesConfigs) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ProtoCatalog.cs:line 39
   at Google.Api.Generator.CodeGenerator.Generate(IReadOnlyList`1 descriptorProtos, IEnumerable`1 filesToGenerate, IClock clock, String grpcServiceConfigPath, IEnumerable`1 commonResourcesConfigPaths, Boolean generateMetadata)+MoveNext() in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/CodeGenerator.cs:line 113
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at Google.Protobuf.Collections.RepeatedField`1.AddRange(IEnumerable`1 values)
   at Google.Protobuf.Collections.RepeatedField`1.Add(IEnumerable`1 values)
   at Google.Api.Generator.Program.GenerateFromProtoc(Stream stdin, Stream stdout) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/Program.cs:line 174
rm: missing operand
Try 'rm --help' for more information.
2021-09-05 08:37:36,082 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Asset.V1:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Asset.V1')' returned non-zero exit status 1.
2021-09-05 08:37:36,109 autosynth [ERROR] > Synthesis failed
2021-09-05 08:37:36,109 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7a79d5a97 feat: add OSConfigZonalService API Committer: @jaiminsh
2021-09-05 08:37:36,137 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1
Switched to branch 'autosynth-Google.Cloud.Asset.V1'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f405ca4c-8311-43fb-ba79-920c3a3cad6d/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

Autosynth is still having trouble generating Google.Cloud.Asset.V1. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Asset.V1 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
osynth-Google.Cloud.Asset.V1
2021-09-05 22:38:15,927 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1
Switched to branch 'autosynth-Google.Cloud.Asset.V1'
2021-09-05 22:38:16,088 autosynth [DEBUG] > Running: git rev-parse --show-toplevel
2021-09-05 22:38:16,091 autosynth [DEBUG] > Running: git log -1 --pretty=%H
2021-09-05 22:38:16,095 autosynth [DEBUG] > Running: git remote get-url origin
2021-09-05 22:38:16,539 autosynth [DEBUG] > Running: git log 9c7eb1f9a78060a15cc309ab51ee0600c2e6f4f7..HEAD --pretty=%H --no-decorate
2021-09-05 22:38:16,543 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9c7eb1f9a78060a15cc309ab51ee0600c2e6f4f7
2021-09-05 22:38:16,546 autosynth [DEBUG] > Running: git log -1 --pretty=%at 31d2d3432d9a0b910d7858eab7720b63445f23b2
2021-09-05 22:38:16,549 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5d91687b183a6f2269ab75344e338f6bc584f5c4
2021-09-05 22:38:16,552 autosynth [DEBUG] > Running: git log -1 --pretty=%at bf57b2c88187c599bf669005d1d2ce80a40780df
2021-09-05 22:38:16,556 autosynth [DEBUG] > Running: git log -1 --pretty=%at b64d30b2b245cb805d1aa328e50258fdff6e628e
2021-09-05 22:38:16,559 autosynth [DEBUG] > Running: git checkout 7a79d5a97cae55043a6a62532e5c4bcfd0512a98
Note: switching to '7a79d5a97cae55043a6a62532e5c4bcfd0512a98'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 7a79d5a97 feat: add OSConfigZonalService API Committer: @jaiminsh
2021-09-05 22:38:16,583 autosynth [DEBUG] > Running: git checkout b64d30b2b245cb805d1aa328e50258fdff6e628e
Note: switching to 'b64d30b2b245cb805d1aa328e50258fdff6e628e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at b64d30b2 chore: regenerate API index
2021-09-05 22:38:16,599 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Asset.V1-5
2021-09-05 22:38:16,603 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1-5
Switched to branch 'autosynth-Google.Cloud.Asset.V1-5'
2021-09-05 22:38:16,627 autosynth [INFO] > Running synthtool
2021-09-05 22:38:16,627 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-09-05 22:38:16,627 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Asset.V1/5/sponge_log.log
2021-09-05 22:38:16,629 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-09-05 22:38:16,868 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py.
On branch autosynth-Google.Cloud.Asset.V1-5
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Asset.V1 for compatibility checking
Generating Google.Cloud.Asset.V1
--gapic_out: System.InvalidOperationException: No resource type with name: 'compute.googleapis.com/Instance' for field ListInventoriesRequest.parent
   at Google.Api.Generator.ProtoUtils.ResourceDetails.LoadResourceReference(MessageDescriptor msgDesc, FieldDescriptor fieldDesc, IReadOnlyDictionary`2 resourcesByUrt, IReadOnlyDictionary`2 resourcesByParentComparison)+MoveNext() in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ResourceDetails.cs:line 288
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Google.Api.Generator.ProtoUtils.ProtoCatalog.<>c__DisplayClass0_2.<.ctor>b__20(FieldDescriptor field) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ProtoCatalog.cs:line 43
   at System.Linq.Enumerable.SelectIListIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector)
   at Google.Api.Generator.ProtoUtils.ProtoCatalog..ctor(String defaultPackage, IEnumerable`1 descs, IEnumerable`1 commonResourcesConfigs) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ProtoCatalog.cs:line 39
   at Google.Api.Generator.CodeGenerator.Generate(IReadOnlyList`1 descriptorProtos, IEnumerable`1 filesToGenerate, IClock clock, String grpcServiceConfigPath, IEnumerable`1 commonResourcesConfigPaths, Boolean generateMetadata)+MoveNext() in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/CodeGenerator.cs:line 113
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at Google.Protobuf.Collections.RepeatedField`1.AddRange(IEnumerable`1 values)
   at Google.Protobuf.Collections.RepeatedField`1.Add(IEnumerable`1 values)
   at Google.Api.Generator.Program.GenerateFromProtoc(Stream stdin, Stream stdout) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/Program.cs:line 174
rm: missing operand
Try 'rm --help' for more information.
2021-09-05 22:38:22,343 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Asset.V1:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Asset.V1')' returned non-zero exit status 1.
2021-09-05 22:38:22,373 autosynth [ERROR] > Synthesis failed
2021-09-05 22:38:22,373 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 7a79d5a97 feat: add OSConfigZonalService API Committer: @jaiminsh
2021-09-05 22:38:22,402 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1
Switched to branch 'autosynth-Google.Cloud.Asset.V1'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/6d4bc063-5099-4abb-bb73-5c8f0ba0ad9b/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

Autosynth is still having trouble generating Google.Cloud.Asset.V1. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Asset.V1 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
d.Asset.V1
2021-09-06 00:58:30,199 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1
Switched to branch 'autosynth-Google.Cloud.Asset.V1'
2021-09-06 00:58:30,378 autosynth [DEBUG] > Running: git rev-parse --show-toplevel
2021-09-06 00:58:30,381 autosynth [DEBUG] > Running: git log -1 --pretty=%H
2021-09-06 00:58:30,385 autosynth [DEBUG] > Running: git remote get-url origin
2021-09-06 00:58:30,852 autosynth [DEBUG] > Running: git log 9c7eb1f9a78060a15cc309ab51ee0600c2e6f4f7..HEAD --pretty=%H --no-decorate
2021-09-06 00:58:30,855 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9c7eb1f9a78060a15cc309ab51ee0600c2e6f4f7
2021-09-06 00:58:30,858 autosynth [DEBUG] > Running: git log -1 --pretty=%at 31d2d3432d9a0b910d7858eab7720b63445f23b2
2021-09-06 00:58:30,861 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5d91687b183a6f2269ab75344e338f6bc584f5c4
2021-09-06 00:58:30,864 autosynth [DEBUG] > Running: git log -1 --pretty=%at bf57b2c88187c599bf669005d1d2ce80a40780df
2021-09-06 00:58:30,866 autosynth [DEBUG] > Running: git log -1 --pretty=%at b64d30b2b245cb805d1aa328e50258fdff6e628e
2021-09-06 00:58:30,869 autosynth [DEBUG] > Running: git checkout c0681507a287f418e3e427a30a0f5aaea634734e
Note: switching to 'c0681507a287f418e3e427a30a0f5aaea634734e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at c0681507a Add google/cloud/extended_operations.proto to Google.LongRunning
2021-09-06 00:58:30,891 autosynth [DEBUG] > Running: git checkout b64d30b2b245cb805d1aa328e50258fdff6e628e
Note: switching to 'b64d30b2b245cb805d1aa328e50258fdff6e628e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at b64d30b2 chore: regenerate API index
2021-09-06 00:58:30,905 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Asset.V1-5
2021-09-06 00:58:30,908 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1-5
Switched to branch 'autosynth-Google.Cloud.Asset.V1-5'
2021-09-06 00:58:30,928 autosynth [INFO] > Running synthtool
2021-09-06 00:58:30,928 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-09-06 00:58:30,929 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Asset.V1/5/sponge_log.log
2021-09-06 00:58:30,930 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-09-06 00:58:31,142 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py.
On branch autosynth-Google.Cloud.Asset.V1-5
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Asset.V1 for compatibility checking
Generating Google.Cloud.Asset.V1
--gapic_out: System.InvalidOperationException: No resource type with name: 'compute.googleapis.com/Instance' for field ListInventoriesRequest.parent
   at Google.Api.Generator.ProtoUtils.ResourceDetails.LoadResourceReference(MessageDescriptor msgDesc, FieldDescriptor fieldDesc, IReadOnlyDictionary`2 resourcesByUrt, IReadOnlyDictionary`2 resourcesByParentComparison)+MoveNext() in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ResourceDetails.cs:line 288
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Google.Api.Generator.ProtoUtils.ProtoCatalog.<>c__DisplayClass0_2.<.ctor>b__20(FieldDescriptor field) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ProtoCatalog.cs:line 43
   at System.Linq.Enumerable.SelectIListIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector, IEqualityComparer`1 comparer)
   at System.Linq.Enumerable.ToDictionary[TSource,TKey,TElement](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector)
   at Google.Api.Generator.ProtoUtils.ProtoCatalog..ctor(String defaultPackage, IEnumerable`1 descs, IEnumerable`1 commonResourcesConfigs) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/ProtoUtils/ProtoCatalog.cs:line 39
   at Google.Api.Generator.CodeGenerator.Generate(IReadOnlyList`1 descriptorProtos, IEnumerable`1 filesToGenerate, IClock clock, String grpcServiceConfigPath, IEnumerable`1 commonResourcesConfigPaths, Boolean generateMetadata)+MoveNext() in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/CodeGenerator.cs:line 113
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at Google.Protobuf.Collections.RepeatedField`1.AddRange(IEnumerable`1 values)
   at Google.Protobuf.Collections.RepeatedField`1.Add(IEnumerable`1 values)
   at Google.Api.Generator.Program.GenerateFromProtoc(Stream stdin, Stream stdout) in /home/kbuilder/.cache/synthtool/gapic-generator-csharp/Google.Api.Generator/Program.cs:line 174
rm: missing operand
Try 'rm --help' for more information.
2021-09-06 00:58:36,264 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Asset.V1:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Asset.V1/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Asset.V1')' returned non-zero exit status 1.
2021-09-06 00:58:36,290 autosynth [ERROR] > Synthesis failed
2021-09-06 00:58:36,290 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at c0681507a Add google/cloud/extended_operations.proto to Google.LongRunning
2021-09-06 00:58:36,316 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Asset.V1
Switched to branch 'autosynth-Google.Cloud.Asset.V1'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/cdd40426-7f47-422d-aab0-e82e99a30039/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

--- a/apis/Google.Cloud.Asset.V1/postgeneration.sh
+++ b/apis/Google.Cloud.Asset.V1/postgeneration.sh
@@ -1,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+# Undo the changes made in pregeneration.sh
+git -C $GOOGLEAPIS checkout google/cloud/asset/v1

--- a/apis/Google.Cloud.Asset.V1/pregeneration.sh
+++ b/apis/Google.Cloud.Asset.V1/pregeneration.sh
@@ -1,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+# Insert an extra import into assets.proto to resolve a resource name.
+# It doesn't matter much where this goes; line 19 is the start of the
+# import block.
+sed -i '19 i import \"google/cloud/osconfig/v1/osconfig_service.proto\";' \
+  $GOOGLEAPIS/google/cloud/asset/v1/assets.proto

